rustest-testing/src/bin/ignored_test.rs

```diff
@@ -1,5 +1,7 @@
 use rustest::{test, *};
 
+pub mod sequential_test;
```

Comment:
You don't need this.

Every rs file in the bin directory will be a detected by cargo and build as a binary.
You don't need to "declare" it there.

Reply:
good catch! i don't know how that happened... maybe my IDE was a bit overeager? if it happens again i'll report it as a bug to them...
(usually i look at the diff and catch these things before committing - not sure how it slipped past me this time, sorry! thanks for spotting it!)